agent/connect/ca/provider_vault.go

```diff
@@ -388,7 +388,7 @@ func (v *VaultProvider) setupIntermediatePKIPath() error {
 	} else {
 		err := v.tuneMountNamespaced(v.config.IntermediatePKINamespace, v.config.IntermediatePKIPath, &mountConfig)
 		if err != nil {
-			return err
+			v.logger.Warn("Could not update intermediate PKI mount settings", "path", v.config.IntermediatePKIPath, "error", err)
```

Comment:
@kyhavlov : Does `err` here contain any messaging explaining that the problem is that the read capability in Vault is missing on the path?

Reply:
Yes - if it's due to permission denied, it'll have that in the error message as well as the endpoint it was trying to access (`../tune` in this case)

Reply:
The log message looks like this: 
```
2022-10-18T11:38:23.609-0700 [WARN]  Could not update intermediate PKI mount settings: path=pki-intermediate/
  error=
  | Error making API request.
  | 
  | URL: POST http://127.0.0.1:16001/v1/sys/mounts/pki-intermediate/tune
  | Code: 403. Errors:
  | 
  | * 1 error occurred:
  | 	* permission denied
  | ```